src/redux/walletconnect.ts

```diff
@@ -658,8 +662,14 @@ const listenOnNewMessages = (walletConnector: WalletConnect) => (
   });
   walletConnector.on('disconnect', error => {
     if (error) {
-      throw error;
+      logger.error(new RainbowError('WC: Error on wc disconnect'), {
+        message: error,
+      });
+
+      // we used to throw, so for parity, return
+      return;
```

Comment:
If it's any, it's this one. This is an event listener, it shouldn't throw because the callsite will never receive an exception.